README.md

```diff
@@ -1,118 +1,43 @@
-
-# How?
-
-Assisting you building a career path in the DevOps domain by driving DevOps knowledge transfer in a study group which develops a real webapp (www.devopsloft.io).
-And, matching you with potentials organizations who seek DevOps engineers with your skillset (technical and personal).
-
-We will study together DevOps, once a week, two hours session frontally using Skype or suchlike (shared lesson).
-The rest of the work is done on your own. You will work on issues assigned to you.
-One can order personal coaching/training.
-
-You are welcomed to attend the program as long as you feel it is valuable for you.
-
-According to your pace, we will set you interviews with potentials organizations.
-
-Recommended skills:
-A degree in Computer Science/Software Development/Industrial Engineering/related technical discipline or equivalent experience (DEV/IT/OPS)
-Ability to understand and write code.
-
-What to bring:
-* yourself
-* your laptop
-
-We will be working with services like: GitHub, AWS and others. Some of those services require payments. Those costs are on your own expense.
-
-# Costs
+## Contributing
 
-Participation is free of charge and commitment
+### Spinning [dev|stage] environment
 
-# Weekly Sessions
-(online or offline)
+##### Global Prerequisites
+* python 3
+* vagrant
+* vagrant plugin: vagrant-env
+* Use `.env.local` file for configuration keys which overrides `.env`
 
-Every Wednesday 6:00pm-9:00pm IDT
-</br>
-</br>
-If you'd like to add a feature or correct a defect, please open a
-[pull request](https://github.com/DevOpsLoft/DevOpsLoft/pulls).</br>
-If you have questions or suggestions, please open an
-[issue](https://github.com/DevOpsLoft/DevOpsLoft/issues).
+#### DEV environment
+##### Prerequisites
+* VirtualBox
 
-And be sure to follow us on [Facebook](https://facebook.com/devopsloft) and [LinkedIn](https://www.linkedin.com/company/devopsloft).
-
-And ... join our [Facebook Group](https://www.facebook.com/groups/512664539127088) and our [Meetup Group](https://www.meetup.com/DevOps-Tel-Aviv/).
-
-# Running the application
-
-## Prerequisites
-
-* Vagrant should be installed in order to launch an environment of the application.
-* AWS account is required in order to run the application in STAGE environment. (see more details in STAGE Environment section)
-* devopsloft.yml file - feature toggle. Feature is released if the value is 'enabled'.
-
-## DEV environment
-
-* The application will be deployed as local VM using VirtualBox
-* Install manually the following plugin: vagrant-env
-```
-vagrant plugin install vagrant-env
-```
-* Create the environment
+#### STAGE environment
+##### Prerequisites
+* AWS account
+* AWS credentials: access key & access secret
+* keypair
+* subnet ID
+* Security Group with inbound ports for SSH (22), HTTP (80) and HTTPS (443)
 
+##### Spin environment
 ```
-vagrant up dev
+./spin.py up [dev|stage]
```

Comment:
Currently does not exists

Reply:
It will be soon enough. I will not merge my PR till it is.